FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_multitask/DemoTasks/MultitaskMQTTExample.c

```diff
@@ -297,7 +308,7 @@ typedef struct ackInfo
  */
 typedef struct subscriptionElement
 {
-    char pTopicFilter[ DEMO_BUFFER_SIZE ];
+    char pTopicFilter[ mqttexampleDEMO_BUFFER_SIZE ];
     uint16_t topicFilterLength;
```

Comment:
FreeRTOS follows a different naming convention for variables.

Reply:
I wanted these structs to have the same field names as MQTT, since a lot of them have a 1:1 correspondence with fields in MQTT's structs, and it made more sense to me if they had the same names. However, I pushed a change to give these fields new names.